A6-client-retries.md

```diff
@@ -287,7 +287,7 @@ Similarly, transparent retries do not count toward the limit of configured RPC a
 
 Both client and server application logic will have access to data about retries via gRPC metadata. Upon seeing an RPC from the client, the server will know if it was a retry, and moreover, it will know the number of previously made attempts. Likewise, the client will receive the number of retry attempts made when receiving the results of an RPC.
 
-The header name for exposing the metadata will be `"grpc-retry-attempts"` to give clients and servers access to the attempt count. The value for this field will be an integer.
+The header name for exposing the metadata will be `"grpc-retry-attempts"` to give clients and servers access to the attempt count. This value represents the number of preceding retry attempts. Thus, it will not be present on the first RPC, will be 1 for the second RPC, and so on. The value for this field will be an integer.
```

Comment:
Given the new semantic, how about changing the header name to include the word "previous"?

Also, I assume that we want to use this for both retries and hedging, but the current name implies only the former.

How about just calling it "grpc-previous-attempts"?